chidioguejiofor/maintenanceApp
# #157941660: Synchronize login page with backend

#### What does this PR do?
- begins consuming API
- configures new heroku app for frontend
- completes functionality for signing up a client
- 

#### Description of Task to be completed?
- user should be able to create account from the site
- user should be able to login from the app 

#### Any background context you want to provide?
- the heroku app for the api are: 
     - **front-end link:** https://chidiebere-maintenance.herokuapp.com/ 

     - **api link: ** chidiebere-maintenance-api.herokuapp.com
#### What are the relevant Github Issues?
- None

#### Questions:
None


#### How should this be tested
- First go to the home page @ chidiebere-maintenance-api.herokuapp.com
- Cilck on the user login button
- A signup page shows 
-  This screenshot shows a sample wrong signup with bad data
 ![bad data](https://user-images.githubusercontent.com/38565349/40733906-229716b4-6437-11e8-9393-d97ccb0d3e29.PNG)


- When you try to update with an existing _username or email_
![existing data](https://user-images.githubusercontent.com/38565349/40733864-0504f760-6437-11e8-9093-8dce925c7fda.PNG)

- When the values are correct
![correctvalues](https://user-images.githubusercontent.com/38565349/40733966-56586304-6437-11e8-8f47-e79f87b1f704.PNG)
 the following page should show up 
![reqeusts page](https://user-images.githubusercontent.com/38565349/40734001-767d67ce-6437-11e8-8b3b-963e42dad3d2.PNG)
